mods/dye/init.lua

```diff
@@ -20,13 +20,104 @@ dye.dyes = {
 	pink		= { name = "Pink", html = "#FF7FFF" },
 }
 
--- Define items
+----
+-- Helper functions.
+----
+
+--[[
+	Convert an html color in the format of #RRGGBB into red, green, and blue values on the range of [0..255].
+]]
+function hex2rgb(hex)
```

Comment:
API / public functions. This should be clear that these are public and require documentation, as well as put in a reasonable namespace (e.g. dye.hex2rgb())